content/news/XXXX-XX-XX-mixxx-2-4-beta-released.md

```diff
@@ -0,0 +1,30 @@
+title: Mixxx 2.3.5 released
+authors: Nikolaus Einhauser
+status: draft
+tags: 2.4, beta, release announcement
+comments: yes
+
+It's been a while, but after almost exactly 3 years, we're proud to announce the
+beta version of the next major release: Mixxx 2.4!  
+Keep your eyes peeled for the stable version of 2.4 being released in August if
+everything goes according to plan. In the meantime, we need help from **YOU**!
+In order to get 2.4 stable, we need lots of people testing this beta release,
+the more, the better. Please [download it](https://mixxx.org/download/#beta)
+and report any bugs you find on [our bugtracker](github.com/mixxxdj/mixxx/issues).
+
+Here's a quick look at the most important changes and new features you can look
+forward to in Mixxx 2.4:
+
+* Support for saving loops as hotcues ("Saved Loops").
+* Save effect chains and then load them into effect units or the QuickEffect
+* Native Apple Silicon support (better performance on M1/M2/etc. devices).
+<!-- how can we summarize sync leader in a single sentence? -->
+* Explicit Sync leader.
+* New controller mapping scripting engine (ES2016 support).
+* GUI performance fixes for macOS (use "all-shaders" waveform types).
+* A bunch of polish for the Library.
+
```

Comment:
Do we want to add cover art download?

Reply:
I found it too small of a change (at least in terms of end user facing features, not in terms of technical effort included) to get its own entry. I figured we could mention that in the feature blog post that addresses the library changes. 

No objections from me though if you insist it to be included.